_raml/apps/products/products.raml

```diff
@@ -34,6 +34,18 @@ get:
         application/json:
           example: !include examples/get_product_response.json
           schema: !include schema/get_product.schema.json
+post:
+  description: Creates a new product.
+  securedBy: [products_write]
+  is: [ locale, currency ]
+  body:
```

Comment:
request body is "object of ..." -> we have to think about a good name..it is like product but with a reduced set of attributes (since some of them cannot set manually and others cannot be set on product creation. In addition there is just a simple price attribute)